plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveOnDataLake.java

```diff
@@ -1605,6 +1605,24 @@ public void testAnalyzePartitionedTableWithCanonicalization()
         assertUpdate("DROP TABLE " + getFullyQualifiedTestTableName(tableName));
     }
 
+    @Test
+    public void testSetColumnType()
+    {
+        String tableName = "test_set_column_type_" + randomNameSuffix();
+        String trinoTableName = getFullyQualifiedTestTableName(tableName);
+        String hiveTableName = getHiveTestTableName(tableName);
+
+        assertUpdate("CREATE TABLE " + trinoTableName + " AS SELECT CAST(123 AS integer) AS col", 1);
+
+        assertUpdate("ALTER TABLE " + trinoTableName + " ALTER COLUMN col SET DATA TYPE bigint");
+
+        assertQuery("SELECT * FROM " + trinoTableName, "VALUES 123");
+        assertThat(onHive("SELECT * FROM " + hiveTableName))
+                .containsPattern("[ |]+123[ |]+");
```

Comment:
onHive returns a String output from a CLI
this isn't really meant to run test queries

let's use product tests setup for that, where we have Hive JDBC and we can run queries normally, without processing textual output of some foreign CLI tool